.github/workflows/build-wheels.yml

```diff
@@ -35,6 +35,7 @@ jobs:
       - name: Build distro
         env:
           ZROK_VERSION: ${{ github.event.release.tag_name }}
+          ZROK_PY_NAME: ${{ vars.ZROK_PY_NAME || null }}
```

Comment:
Get the PyPi module name from a var if exists, else set null. There's a counterpart change in setup.py to handle both cases.